src/betterproto/__init__.py

```diff
@@ -601,6 +603,43 @@ def _get_cls_by_field(
         return field_cls
 
 
+HybridT = TypeVar("HybridT")
+
+
+class hybridmethod(Generic[HybridT]):
+    """Hybrid method decorator.
+
+    Can be used to have both classmethod and instance method with the same name and
+    execute the right one depending on context.
+
+    Source: https://stackoverflow.com/a/28238047
+    """
+
+    def __init__(
+        self,
+        fclass: Type[HybridT],
+        finstance: Optional[HybridT] = None,
+        doc: Optional[str] = None,
+    ):
+        self.fclass = fclass
+        self.finstance = finstance
```

Comment:
Not sure why these are f prefixed. Please remove the prefix and name them something like owner and instance as is commonly  done with the descriptor protocol